{"created": 1653067854.1182623, "duration": 0.36807703971862793, "exitcode": 1, "root": "C:\\Python_Projects\\pytest-notifier-bot\\pytest", "environment": {"Python": "3.10.4", "Platform": "Windows-10-10.0.18363-SP0", "Packages": {"pytest": "7.1.2", "py": "1.11.0", "pluggy": "1.0.0"}, "Plugins": {"json-report": "1.5.0", "metadata": "2.0.1"}}, "summary": {"failed": 2, "passed": 3, "total": 5, "collected": 5}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "ch1/test_one.py", "type": "Module"}, {"nodeid": "ch1/test_three.py", "type": "Module"}, {"nodeid": "ch1/test_two.py", "type": "Module"}]}, {"nodeid": "ch1/test_one.py", "outcome": "passed", "result": [{"nodeid": "ch1/test_one.py::test_passing", "type": "Function", "lineno": 0}, {"nodeid": "ch1/test_one.py::test_passing2", "type": "Function", "lineno": 3}]}, {"nodeid": "ch1/test_three.py", "outcome": "passed", "result": [{"nodeid": "ch1/test_three.py::test_defaults", "type": "Function", "lineno": 6}, {"nodeid": "ch1/test_three.py::test_member_access", "type": "Function", "lineno": 12}]}, {"nodeid": "ch1/test_two.py", "outcome": "passed", "result": [{"nodeid": "ch1/test_two.py::test_failing", "type": "Function", "lineno": 0}]}], "tests": [{"nodeid": "ch1/test_one.py::test_passing", "lineno": 0, "outcome": "failed", "keywords": ["test_passing", "ch1/test_one.py", "pytest"], "setup": {"duration": 0.0004890999989584088, "outcome": "passed"}, "call": {"duration": 0.0008120000129565597, "outcome": "failed", "crash": {"path": "C:\\Python_Projects\\pytest-notifier-bot\\pytest\\ch1\\test_one.py", "lineno": 2, "message": "assert (1, 2, 1) == (1, 2, 3)\n  At index 2 diff: 1 != 3\n  Use -v to get more diff"}, "traceback": [{"path": "pytest\\ch1\\test_one.py", "lineno": 2, "message": "AssertionError"}], "longrepr": "def test_passing():\n>       assert (1, 2, 1) == (1, 2, 3)\nE       assert (1, 2, 1) == (1, 2, 3)\nE         At index 2 diff: 1 != 3\nE         Use -v to get more diff\n\npytest\\ch1\\test_one.py:2: AssertionError"}, "teardown": {"duration": 0.00031619996298104525, "outcome": "passed"}}, {"nodeid": "ch1/test_one.py::test_passing2", "lineno": 3, "outcome": "failed", "keywords": ["test_passing2", "ch1/test_one.py", "pytest"], "setup": {"duration": 0.0004029000410810113, "outcome": "passed"}, "call": {"duration": 0.0011307999957352877, "outcome": "failed", "crash": {"path": "C:\\Python_Projects\\pytest-notifier-bot\\pytest\\ch1\\test_one.py", "lineno": 5, "message": "assert (1, 444, 1) == (1, 2, 3)\n  At index 1 diff: 444 != 2\n  Use -v to get more diff"}, "traceback": [{"path": "pytest\\ch1\\test_one.py", "lineno": 5, "message": "AssertionError"}], "longrepr": "def test_passing2():\n>       assert (1, 444, 1) == (1, 2, 3)\nE       assert (1, 444, 1) == (1, 2, 3)\nE         At index 1 diff: 444 != 2\nE         Use -v to get more diff\n\npytest\\ch1\\test_one.py:5: AssertionError"}, "teardown": {"duration": 0.0002929000183939934, "outcome": "passed"}}, {"nodeid": "ch1/test_three.py::test_defaults", "lineno": 6, "outcome": "passed", "keywords": ["test_defaults", "ch1/test_three.py", "pytest"], "setup": {"duration": 0.00027540000155568123, "outcome": "passed"}, "call": {"duration": 0.00029250001534819603, "outcome": "passed"}, "teardown": {"duration": 0.00020750006660819054, "outcome": "passed"}}, {"nodeid": "ch1/test_three.py::test_member_access", "lineno": 12, "outcome": "passed", "keywords": ["test_member_access", "ch1/test_three.py", "pytest"], "setup": {"duration": 0.00021039997227489948, "outcome": "passed"}, "call": {"duration": 0.00027630000840872526, "outcome": "passed"}, "teardown": {"duration": 0.0002106999745592475, "outcome": "passed"}}, {"nodeid": "ch1/test_two.py::test_failing", "lineno": 0, "outcome": "passed", "keywords": ["test_failing", "ch1/test_two.py", "pytest"], "setup": {"duration": 0.00023150001652538776, "outcome": "passed"}, "call": {"duration": 0.00021049997303634882, "outcome": "passed"}, "teardown": {"duration": 0.00042260007467120886, "outcome": "passed"}}]}